tests/BinaryFormatterTests/WhenWorkingWith_Classes.cs

```diff
@@ -55,5 +146,31 @@ public void CanWorkWith_ClasseWithCustomCtor_WithProperties_WithPublicSetter()
             Assert.Equal(before.Int, after.Int);
             Assert.Equal(before.Double, after.Double);
         }
+
+        [Fact]
+        public void CanWorkWith_ClassesWithoutCustomCtor_WithVirtualProperties_WithPublicSetter()
```

Comment:
@lukasz-pyrzyk Did you means to make test more simple and decline hidden static collection?